src/server_manager/web_app/digitalocean_app/connect_account_app.ts

```diff
@@ -0,0 +1,164 @@
+/*
+  Copyright 2018 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+import '@polymer/polymer/polymer-legacy.js';
+import '@polymer/iron-pages/iron-pages.js';
+import '../ui_components/outline-step-view.js';
+
+import {css, customElement, html, LitElement, property} from 'lit-element';
+
+import {RestApiSession} from '../../cloud/digitalocean_api';
+import {makePublicEvent} from '../../infrastructure/events';
+import {LocalStorageRepository} from '../../infrastructure/repository';
+import {getSentryApiUrl} from '../../infrastructure/sentry';
+import * as account from '../../model/account';
+import {AccountModelFactory} from '../../model/account';
+import * as cloud_provider from '../../model/cloud_provider';
+import {DigitalOceanAccount} from '../../model/digitalocean_account';
+import {AppSettings} from '../app';
+import {COMMON_STYLES} from '../ui_components/cloud-install-styles';
+import {OutlineNotificationManager} from '../ui_components/outline-notification-manager';
+
+@customElement('digital-ocean-connect-account-app')
+export class DigitalOceanConnectAccountApp extends LitElement implements
+    AccountModelFactory<DigitalOceanAccount> {
+  @property({type: Function}) localize: Function;
+  @property({type: Object}) appSettings: AppSettings = null;
```

Comment:
appSettings and accountRepository can be replaced by a rich DigitalOceanCloud object. See below.